client/src/Main/Home/Home.js

```diff
@@ -7,12 +7,19 @@ import SimpleCard from './SimpleCard/SimpleCard';
 import { saveSettings } from '../Settings/SettingsActions';
 import { resetMonitor } from '../Monitor/MonitorActions';
 import BacklogAutocomplete from './BacklogAutocomplete';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+} from '@material-ui/core';
 
 const getTicketPointsFromName = name => {
   const regex = /\((\?|\d+\.?,?\d*)\)/m;
   const points = regex.exec(name);
 
-  return points ? parseInt(points[1]) : 0;
+  return points ? parseInt(points[1]) : -1;
```

Comment:
I am not very confortable with having a negative point number in bdd. It will be transformed to -1*3600*1000 ms for `allocatedTime`